jrportfolio.css

```diff
@@ -152,8 +153,9 @@ html {
 .box-1container {
 	float: left;
 	width: 33.3%;
-	height: 340px;
+	height: auto;
 	padding: 0 50px;
+	margin-bottom:0.5rem;
```

Comment:
Make sure there is a space between the property and value on all rules, eg: margin-bottom: 0.5rem;

Reply:
Done